apps/csv-to-json/src/App.jsx

```diff
@@ -101,7 +101,8 @@ const CopyButton = styled.button`
 //=========================================
 
 function Component() {
-  const [csvData, setCSVData] = useState();
+  const defaultData = "FirstName;LastName\nJohn;Doe"
```

Comment:
@Sync271 I believe we should use `;`, `,`. Rest looks good

Reply:
Yeah, my bad!